src/group/mls_group/create_commit.rs

```diff
@@ -117,10 +118,20 @@ impl MlsGroup {
             &joiner_secret,
             apply_proposals_values.invitation_list,
             &provisional_tree,
+            &apply_proposals_values.presharedkeys,
         )?;
 
-        // TODO #141: Implement PSK
-        let mut key_schedule = KeySchedule::init(ciphersuite, joiner_secret, None);
+        // Create key schedule
+        let mut key_schedule = KeySchedule::init(
+            ciphersuite,
+            joiner_secret,
+            psk_output(
```

Comment:
The reason we have this is because we don't know what the proposals contain beforehand, especially those that we received from other members. We would have to parse them ahead of time to know exactly what PSKs to pass here.